src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -103,30 +105,40 @@ void tabulate() {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     if (config != null && config.validate() && setUpLogging(config)) {
-      if (config.isSequentialMultiSeatEnabled()) {
-        int numWinners = config.getNumberOfWinners();
-        // temporarily set config to single-seat so we can run sequential elections
-        config.setNumberOfWinners(1);
-        while (config.getSequentialWinners().size() < numWinners) {
-          Set<String> newWinnerSet = runTabulationForConfig(config);
-          assert (newWinnerSet.size() == 1);
-          String newWinner = (String) newWinnerSet.toArray()[0];
-          config.setCandidateExclusionStatus(newWinner, true);
-          config.addSequentialWinner(newWinner);
-        }
-        // revert config to original state
-        config.setNumberOfWinners(numWinners);
-        for (String winner : config.getSequentialWinners()) {
-          config.setCandidateExclusionStatus(winner, false);
+        if (config.isSequentialMultiSeatEnabled()) {
+          int numWinners = config.getNumberOfWinners();
+          // temporarily set config to single-seat so we can run sequential elections
+          config.setNumberOfWinners(1);
+          while (config.getSequentialWinners().size() < numWinners) {
+            // Read cast vote records and precinct IDs from CVR files
+            List<CastVoteRecord> castVoteRecords = parseCastVoteRecords(config, precinctIDs);
+            if (castVoteRecords == null) {
+              Logger.log(Level.SEVERE, "Aborting tabulation due to cvr errors!");
```

Comment:
Here and also in other log messages in this PR: you use "cvr", but you also use "cast vote record" in some other log messages. I recommend doing a find / replace to consistently stick with the latter.